src/loan-comparison/index.html

```diff
@@ -1,5 +1,6 @@
 {% extends "base.html" %}
 {% set active_page = "loan-comparison" %}
+{% set page_js = "loan-comparison" %}
```

Comment:
@amymok Thanks for catching that! I added the page variable to the prepare worksheets page.